src/main/resources/org/sagebionetworks/web/client/widget/evaluation/EvaluationEditorModalViewImpl.ui.xml

```diff
@@ -75,7 +75,7 @@
               <b:FormLabel text="Number of Rounds" />
               <w:NumberBox ui:field="numberOfRoundsField" />
             </b:FormGroup>
-            <bh:Div addStyleNames="light-border padding-10">
+            <bh:Div addStyleNames="padding-10">
```

Comment:
I have not been able to visually inspect this, as this UI is only shown for old challenges that use the "quota" system